infra.bs

```diff
@@ -693,9 +693,9 @@ following steps return true:
  </li>
 </ol>
 
-<p>"<var>input</var> <dfn export for="byte sequence">starts with</dfn> <var>potentialPrefix</var>
-can be used as a synonym for "<var>potentialPrefix</var> is a <a for="byte sequence">prefix</a> of
-<var>input</var>".
+<p>"<var>input</var> <dfn export for="byte sequence" lt="start with|starts with">starts with</dfn>
+<var>potentialPrefix</var> can be used as a synonym for "<var>potentialPrefix</var> is a
+<a for="byte sequence">prefix</a> of <var>input</var>".
```

Comment:
_desperately trying to not complain about hard-wrapping again_

Reply:
I tend to agree that we should change this (ideally we'd have automatic formatting). The current setup is such a time waste during review and makes it easier to overlook the bigger problems.